development-defaults.ini

```diff
@@ -107,7 +107,7 @@ no_shirt     = 0
 'MAGFest 13 tshirt'           = 15
 'MAGFest 12 tshirt'           = 10
 'MAGFest hoodie'              = 30
```

Comment:
I actually need to check with Ryan about getting an updated list of swag and prices.